### PR TITLE
added solution of Palindrome Number in Python

### PR DESCRIPTION
**Pull request by FahimFBA**
Verdict: Accepted

### CODE REVIEW

File: Algorithms/Easy/9_PalindromeNumber/Solution.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Solution(object):
+    def isPalindrome(self, x): # type x : int & return type: bool
```
Comment (Tahanima) [inline Algorithms/Easy/9_PalindromeNumber/Solution.py:2]: Fix the line - `def isPalindrome(self, x): # type x : int & return type: bool` like it is presented in LeetCode editor.

File: Algorithms/Easy/9_PalindromeNumber/Solution.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+        if x < 0:
+            return False
```
Comment (Tahanima) [inline Algorithms/Easy/9_PalindromeNumber/Solution.py:4]: Add a blank line after `return False`.

File: Algorithms/Easy/9_PalindromeNumber/Solution.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+        b =int(str(x) [::-1])
```
Comment (Tahanima) [inline Algorithms/Easy/9_PalindromeNumber/Solution.py:5]: Change this code to 
```py
b = int(str(x) [::-1])
```

File: Algorithms/Easy/9_PalindromeNumber/Solution.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+        if b == x:
+            return True
```
Comment (Tahanima) [inline Algorithms/Easy/9_PalindromeNumber/Solution.py:7]: Add a blank line after `return True`.

**Review by Tahanima — CHANGES_REQUESTED** [review]
Hi @FahimFBA 
Please resolve the requested changes, then your pr will be ready for merge. 

Cheers 🎉

File: Algorithms/Easy/9_PalindromeNumber/Solution.py (hunk 3 above)
Comment (FahimFBA) [reply Algorithms/Easy/9_PalindromeNumber/Solution.py]: > Hi @FahimFBA
> Please resolve the requested changes, then your pr will be ready for merge.
> 
> Cheers 🎉

Hi @Tahanima , I have resolved the requested changes. 😀

**Review by Tahanima — APPROVED** [review]
Great work! 💯

**Comment by FahimFBA** [comment]
> Great work! 💯

Thanks a lot 😀

**Pull request merged**